templates/elementary-OS-app/data/hello-world.desktop.in

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Hello World
+GenericName=Hello World App
+Comment=Proves that we can use Vala and Gtk
+Categories=Utility;Education;
+Exec=com.github.yourusername.yourrepositoryname
+Icon=com.github.yourusername.yourrepositoryname
+Terminal=false
+Type=Application
+Keywords=Hello;World;Example;
```

Comment:
I think a new type of variable (array) might have to be introduced here, so that the user can specify keywords for their application.